.gitignore

```diff
@@ -1,3 +1,6 @@
 dist/
 *.out
 *.test
+output/*
+testdata/*
```

Comment:
I don't think these are needed by this change or in general. Am I missing a need for them?

Reply:
From my perspective, the current project does not have a sufficiently detailed build and test environment (not just Unittest, but also requiring some original Thrift files as test inputs), which is not very friendly to contributors. 

However, I also agree that adding them in such a hasty manner may not be the best approach. If there is time, we could consider a more comprehensive approach to setting up the build and test environment (such as starting with the addition of a Makefile). For now, I will remove the folders from `.gitignore`

Reply:
That sounds good, and I agree those would be good improvements. I previously ran this tool regularly against the (very large!) Pinterest Thrift schema repository in addition to the unit tests, but I think it would be great to have a more comprehensive integration test suite in the open source repository itself.